packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -1782,12 +1807,20 @@ export enum MachineImageType {
   BOTTLEROCKET
 }
 
-const GPU_INSTANCETYPES = ['p2', 'p3', 'g4'];
-const INFERENTIA_INSTANCETYPES = ['inf1'];
+export const GPU_INSTANCETYPES = ['p2', 'p3', 'g2', 'g3', 'g4'];
```

Comment:
Move to a source file that is not exported in `index.ts` so these are not part of the public api of the module.